tests/unit/pages/Editor.spec.js

```diff
@@ -171,31 +171,26 @@ describe("Editor.vue", () => {
       },
     });
     // `getPlio` inside services/API/Plio.js should've been called
-    // 2 `GET` requests are made
-    expect(mockAxios.get).toHaveBeenCalledTimes(2);
+    // 1 `GET` request is made
+    expect(mockAxios.get).toHaveBeenCalledTimes(1);
     expect(mockAxios.get).toHaveBeenCalledWith(`/plios/${plioId}`);
-    expect(mockAxios.get).toHaveBeenCalledWith("/items/", {
-      params: { plio: `${plioId}` },
-    });
 
     // using some pre-defined dummy data to return as a fake response
     // from the fake API call
     let plioResponse = dummyDraftPlio;
-    let itemResponse = dummyItemsCopy;
 
-    // resolve the two `GET` requests waiting in the queue
+    // resolve the `GET` request waiting in the queue
     // using the fake response data
     mockAxios.mockResponse(plioResponse, mockAxios.queue()[0]);
-    mockAxios.mockResponse(itemResponse, mockAxios.queue()[1]);
 
     // wait until the DOM updates after promises resolve
     await flushPromises();
 
     // use `wrapper.vm.__` to access the updated data variables inside the component
     expect(wrapper.vm.loadedPlioDetails.items).toStrictEqual(
-      dummyItemsCopy.data
```

Comment:
There are many other places where `dummyItemsCopy` is being used - do search for it in the file and you'll find it - we should be replacing all of them right?